helios-tools/src/main/java/com/spotify/helios/cli/command/RollingUpdateCommand.java

```diff
@@ -141,10 +147,13 @@ protected int runWithJobId(final Namespace options, final HeliosClient client,
     final long rolloutTimeout = options.getLong(rolloutTimeoutArg.getDest());
     final boolean migrate = options.getBoolean(migrateArg.getDest());
     final boolean overlap = options.getBoolean(overlapArg.getDest());
+    final int failureThreshold = options.getInt(failureThresholdArg.getDest());
 
     checkArgument(timeout > 0, "Timeout must be greater than 0");
     checkArgument(parallelism > 0, "Parallelism must be greater than 0");
     checkArgument(rolloutTimeout > 0, "Rollout timeout must be greater than 0");
+    checkArgument(failureThreshold > 0 && failureThreshold < 100,
```

Comment:
Change semantics so that 0 and 100 are allowed values, with 0 meaning any failures cause the rolling update to fail.